features/step_definitions/cli_steps.js

```diff
@@ -6,145 +6,149 @@ import rimraf from 'rimraf'
+  if (cleansingNeeded) {
+    try {
+      rimraf.sync(tmpDir)
+    } catch (e) {} // eslint-disable-line no-empty
+    cleansingNeeded = false
   }
+}
 
-  this.Given(/^a file named "(.*)" with:$/,
-    (filePath, fileContent, callback) => {
-      cleanseIfNeeded()
-      const absoluteFilePath = `${tmpDir}/${filePath}`
-      const filePathSegments = absoluteFilePath.split('/')
-      filePathSegments.pop()
-      const dirName = joinPathSegments(filePathSegments)
-
-      // replace placeholder for generate unique ids
-      const replacedFileContent = fileContent
-        .replace(/<id-(\w)>/gi, (match, g1) => uniqueId(g1))
-
-      mkdirp(dirName, (err1) => {
-        if (err1) throw new Error(err1)
-
-        fs.writeFile(absoluteFilePath, replacedFileContent, (err2) => {
-          if (err2) throw new Error(err2)
-          callback()
-        })
-      })
-    })
+function normalizeText (text) {
+  return text
+    // .replace(/\033\[[0-9;]*m/g, '')
+    .replace(/\r\n|\r/g, '\n')
+    .replace(/^\s+/g, '')
+    .replace(/\s+$/g, '')
+    .replace(/[ \t]+\n/g, '\n')
+}
 
-  this.When(/^I run `sphere(| .+)`$/, (args, callback) => {
-    const initialCwd = process.cwd()
-    process.chdir(tmpDir)
-
-    const runtimePath = joinPathSegments([ baseDir, 'bin', 'sphere' ])
-    const command = runtimePath + args
-    exec(command, (error, stdout, stderr) => {
-      this.lastRun = { error, stdout, stderr }
-      process.chdir(initialCwd)
-      cleansingNeeded = true
-      callback()
-    })
-  })
+function getAdditionalErrorText (lastRun) {
+  return `Additional error:\n'${lastRun.error}'.\n` +
+    `stderr:\n'${lastRun.stderr}'.`
+}
 
-  this.When(
-    /^I run with accessToken `sphere(| .+)`$/,
-    { timeout: 60 * 1000 },
-    (args, callback) => {
-      loadCredentials().then((credentials) => {
-        const oauth2 = new OAuth2({ config: credentials })
-        oauth2.getAccessToken((err, response, body) => {
-          if (err)
-            throw new Error(`Error occurred when fetching accessToken ${err}`)
-          const accessToken = body.access_token
-          const initialCwd = process.cwd()
-          process.chdir(tmpDir)
-
-          const runtimePath = joinPathSegments([ baseDir, 'bin', 'sphere' ])
-          const command = `${runtimePath} ${args} --accessToken ${accessToken}`
-          const env = Object.assign({}, process.env, {
-            SPHERE_CLIENT_ID: undefined,
-            SPHERE_CLIENT_SECRET: undefined,
-          })
-          const options = { env }
-          exec(command, options, (error, stdout, stderr) => {
-            this.lastRun = { error, stdout, stderr }
-            process.chdir(initialCwd)
-            cleansingNeeded = true
-            callback()
-          })
-        })
+Given(
+  /^a file named "(.*)" with:$/,
+  (filePath, fileContent, callback) => {
+    cleanseIfNeeded()
+    const absoluteFilePath = `${tmpDir}/${filePath}`
+    const filePathSegments = absoluteFilePath.split('/')
```

Comment:
We can use `path` module which will handle also a difference between path separator on Windows vs *nix systems.